KIF.xcodeproj/project.pbxproj

```diff
@@ -1442,7 +1462,7 @@
 				DYLIB_CURRENT_VERSION = 1;
 				DYLIB_INSTALL_NAME_BASE = "@rpath";
 				ENABLE_BITCODE = NO;
-				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				ENABLE_STRICT_OBJC_MSGSEND = NO;
```

Comment:
I don't think we'll be able to merge with this disabled. Looking in CocoaPods, this is turned on by default, so it will not pass a `pod lib lint`.